charts/harvester-cloud-provider/dependency_charts/kube-vip/templates/daemonset.yaml

```diff
@@ -53,7 +53,7 @@ spec:
         envFrom:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-        image: {{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}
+        image: {{ template "system_default_registry" . }}{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}
```

Comment:
You need to add this template back to the subchart: https://github.com/harvester/charts/blob/master/charts/harvester-cloud-provider/dependency_charts/kube-vip/templates/_helpers.tpl

Reply:
Updated it. Thanks.